doc/zeek.rst

```diff
@@ -158,8 +158,10 @@ properties are supported:
         Specifies one or more well-known ports for which you want Zeek to
         automatically activate your analyzer with corresponding
         connections. Each port must be specified in Spicy's :ref:`syntax
-        for port constants <type_port>` (e.g., ``80/tcp``). The ports'
-        transport protocol must match that of the analyzer.
+        for port constants <type_port>` (e.g., ``80/tcp``), or as a port range
+        ``PORT_0-PORT_1`` where start and end port are port constants forming a
```

Comment:
Suggest one more tweak to from ``PORT_0-PORT_1`` back to ``PORT_START-PORT_END`` (because `PORT_0/1` is already used differently a few lines up.